dipy/data/fetcher.py

```diff
@@ -39,9 +39,9 @@ class FetcherError(Exception):
 
 def _log(msg):
```

Comment:
Do we really want to keep this function? Since you are using `logging` module, you should delete it.

Reply:
It's useful as shorthand. I just pushed a clarification of the docstring.